src/lbp.jl

```diff
@@ -134,7 +134,7 @@ function _create_descriptor{T<:FixedPointNumbers.UFixed}(img::AbstractArray{Gray
     descriptor
 end
 
-function create_descriptor{T<:FixedPointNumbers.UFixed}(img::AbstractArray{Gray{T}, 2}, yblocks::Integer = 4, xblocks = 4; lbp_type::Function = lbp, args...)
+function create_descriptor{T<:FixedPointNumbers.Normed}(img::AbstractArray{Gray{T}, 2}, yblocks::Integer = 4, xblocks = 4; lbp_type::Function = lbp, args...)
```

Comment:
Not necessary to change, but one can get rid of the `FixedPointNumbers` scoping and just use `Normed` directly. (As long as you've said `using FixedPointNumbers` rather than `import FixedPointNumbers`.)